File: OfficinaWebApp/src/main/java/com/officina/entity/DataModel.java
package com.officina.entity;

public class DataModel {
    private Pratica pratica;
    private Vettura vettura;
    private Persona persona;

    public DataModel() {
    }

    public Pratica getPratica() {
        return pratica;
    }

    public void setPratica(Pratica pratica) {
        this.pratica = pratica;
    }

    public Vettura getVettura() {
        return vettura;
    }

    public void setVettura(Vettura vettura) {
        this.vettura = vettura;
    }

    public Persona getPersona() {
        return persona;
    }

    public void setPersona(Persona persona) {
        this.persona = persona;
    }
}

File: fronEntOfficina/src/app/model/data.model.ts
import { Pratica } from './pratica.model';
import { Vettura } from './vettura.model';
import { Persona } from './persona.model';

export class dataModel {
  pratica: Pratica;
  vettura: Vettura;
  persona: Persona;

  constructor(pratica: Pratica, vettura: Vettura, persona: Persona) {
    this.pratica = pratica;
    this.vettura = vettura;
    this.persona = persona;
  }
}
